packages/core/botpress/src/web/components/Layout/index.jsx

```diff
@@ -50,9 +51,11 @@ class Layout extends React.Component {
       'bp-container': hasHeader
     })
 
+    const toastCodes = ['nlu-sync', 'qna-save']
```

Comment:
This is OK for now but it means community modules won't be able to use this mechanism without us changing the core. Do we really want this? The codes are not used anywhere else other than to filter the events as I can see

Reply:
@emirotin Good remark! How you look on this. We can save `toastCodes` in `kvs or db` and we will fetch `toastCodes` in `components/Util/BackendToast.js` from server?